peerconnection.go

```diff
@@ -330,10 +330,6 @@ func (pc *PeerConnection) checkNegotiationNeeded() bool { //nolint:gocognit
 		return true
 	}
 
-	if pc.sctpTransport == nil {
```

Comment:
`sctpTransport` is never nil. If something is depending on this behavior I would like to know